tests/gpflow/conditionals/test_broadcasted_conditionals.py

```diff
@@ -65,7 +65,7 @@ def test_conditional_broadcasting(full_cov: bool, white: bool, conditional_type:
     and conditional should broadcast over the [...].
     """
     q_mu = np.random.randn(Data.M, Data.Dy)
-    q_sqrt = np.tril(np.random.randn(Data.Dy, Data.M, Data.M), -1)
+    q_sqrt: AnyNDArray = np.tril(np.random.randn(Data.Dy, Data.M, Data.M), -1)
```

Comment:
I'm really curious why these have suddenly started being necessary. What error do you get without the annotations?

Reply:
> I'm really curious why these have suddenly started being necessary. What error do you get without the annotations?

```
tests/gpflow/conditionals/test_broadcasted_conditionals.py:68: error: Need type annotation for "q_sqrt"
```
:shrug: